client/src/extension.ts

```diff
@@ -44,8 +45,69 @@ export function activate(context: vscode.ExtensionContext) {
     clientOptions
   );
 
-  client.start();
-}
+  client.start(); 
+
+  const endpoint = process.env['AZURE_OPENAI_ENDPOINT'];
+  const azureApiKey = process.env['AZURE_OPENAI_KEY'];
+
+  const systemMessage = 'You are an AI assistant that helps IT administrators generate JSON documents based on the related '+
+	'settings that the user requests. You will never include any chat text in your response, just a JSON file only.';
+
+  vscode.commands.registerCommand('intern-input-test.DCGenerator', async () => {
+    const userInput = await vscode.window.showInputBox({
+      placeHolder: 'Enter setting here',
+      prompt: 'Enter in desired configuration setting for DC document (Ex: Network Access)'
+    });
+
+    if (userInput === undefined) {
+      vscode.window.showErrorMessage('Not a valid input');
+      return;
+    }
+    else if (userInput === '') {
+      vscode.window.showErrorMessage('Setting configuation is required to execute this action');
+    }
+    else {
+      vscode.window.showInformationMessage('Generating DC Doc with: ' + userInput);
+      const userMessage = 'You will need to generate a new desired configuration JSON document based on the information present in the included list. ' +
+				'The desired configuration JSON will have an "OsConfiguration" object, with a "Document" Object and a "Scenario" array ' +
+				'inside of it. The "Document" object is composed of a "schemaversion", "id", "version", "context", and "scenario" variable. ' +
+				'The default values for these variables are included in the document. The "Scenario" array is composed of an object that has ' +
+				'a "name" variable, a "schemaversion" variable, a "action" variable, and an object that has the same name as the configuration scenario. ' +
+				'This object is composed of the relevant settings from the document I am giving you. Inside this object, you should only include ' +
+				'the setting name and default value of settings that relate to ' + userInput + ' only. All values in the JSON should be in string format. ' +
+				'Construct a JSON based on this information and the provided configuration information. Remember, you will never include any chat text in your response, just JSON data only. ' +
+				'Also, you should only include settings that are related to ' + userInput + '. Do not include unrelated settings in the JSON. \n --- \n LIST \n ';
+
+      const scdList: string = 'Configuration Scenario: ASHCIApplianceSecurityBaselineConfig; Schemaversion: 1.0; ID: 64789; version: 1.0; context: device. Action: set.   \n Settings: \n' +
```

Comment:
This particular SCD list is temporary. API prompts are still being experimented with/tweaked to determine what is the most efficient way to obtain a valid DC document from the AI model.  In the future, it may be best to load from a directory, but for now the SCD list will remain static.